integration/chaincode/simple/cmd/main.go

```diff
@@ -48,6 +48,10 @@ func handleSigTerm() {
 }
 
 func main() {
+	if os.Getenv("DEVMODE_ENABLED") != "" {
+		fmt.Println("starting up in devmode...")
+	}
+
```

Comment:
If this is being used as the readiness check, shouldn't it happen after starting the go routine that calls `shim.Start`?  Neither is perfect but it seems closer to what we'd expect.

Reply:
Done